doc/guides/writing-tests.md

```diff
@@ -325,6 +325,11 @@ Next add the test to the `sources` in the `cctest` target in node.gyp:
   ...
 ],
 ```
+Note that the only sources that should be included in the cctest are actual
```

Comment:
"cctest target"?

Reply:
Yeah, that should be cctest target, I'll update this.